llama_index/extractors/utils.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+from typing import Any, Coroutine, List
+
+DEFAULT_NUM_WORKERS = 4
+
+
+def get_asyncio_module(show_progress: bool = False) -> Any:
```

Comment:
those util functions could be used in other places tbh. not only for metadata extract.
since it enables multi works run async. is there any other better places we could put it so that other ayncs workers kind of job can reuse it

Reply:
yea good point, will do